app/assets/stylesheets/application.css.scss

```diff
@@ -187,17 +148,47 @@ body h1, h2, h3, h4, h5, h6, .h1, .h2, .h3, .h4, .h5, .h6 {
 
 .main_panel {
   padding-left: 150px;
+  .thought-teasers{
```

Comment:
Rule declaration should be preceded by an empty line<br>Opening curly brace `{` should be preceded by one space